tests/sdk/v2/api/test_config.py

```diff
@@ -93,7 +93,7 @@ def test_returns_true_for_matching_configs(self, config):
                 name=config.name,
                 bypass_factory_methods=True,
             )
-            test_config.uri = config.uri
+            test_config.uri = config.uri  # type: ignore
```

Comment:
Seeing all these type errors when accessing config fields inspired me to reconsider another `RuntimeConfig` refactor. I'm still figuring out the details, but https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/58 was the prelude.

Reply:
@zapatacomputing/sdk-team